src/FSharp.Data.Fred/JsonApi.fs

```diff
@@ -6,6 +6,73 @@ open FSharp.Data
 
 type SearchType = FullText | SeriesId
 
+type SearchOrder = 
+    | SearchRank
+    | SeriesIdOrder
```

Comment:
Nice use of discriminated unions to model the data. This is a lot clearer than using strings that you could accidentally mistype.